recipe/build-mpi.sh

```diff
@@ -12,12 +12,13 @@ export CC=$(basename "$CC")
 export CXX=$(basename "$CXX")
 export FC=$(basename "$FC")
 
-if [ $(uname) == Darwin ]; then
+./autogen.pl --force
+
+if [ "$target_platform" == osx-* ]; then
```

Comment:
I think we want double-brackets and no quotes here?

Reply:
Thanks